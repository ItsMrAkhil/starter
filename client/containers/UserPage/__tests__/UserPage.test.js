import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import { StaticRouter } from 'react-router-dom';

import { UserPage } from '../index';

describe('<UserPage />', () => {
  test('renders', () => {
    const mockData = {
      match: {
        params: { user: 'Akhil' },
      },
      length: 5,
      onFetchName: () => ({}),
    };

    const tree = ReactTestRenderer.create(
      <StaticRouter context={{}} >
        <UserPage {...mockData} />
      </StaticRouter>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
