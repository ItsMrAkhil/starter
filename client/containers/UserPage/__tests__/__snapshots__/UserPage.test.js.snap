// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserPage /> renders 1`] = `
Array [
  <h2>
    Welcome 
    Akhil
  </h2>,
  <code>
    Below Name Length Comes from server
  </code>,
  <br />,
  <br />,
  <span>
    Length : 
    5
     
  </span>,
  " ",
  <br />,
  <br />,
  <a
    href="/"
    onClick={[Function]}
  >
    <button>
       Back to Home
    </button>
  </a>,
]
`;
